code/datums/martial/_martial.dm

```diff
@@ -9,6 +9,8 @@
 	var/help_verb
 	var/allow_temp_override = TRUE //if this martial art can be overridden by temporary martial arts
 	var/smashes_tables = FALSE //If the martial art smashes tables when performing table slams and head smashes
+	var/mob/living/holder //owner of the martial art
```

Comment:
Make this a weakref, otherwise you'll force hard dels of things with martial arts.